src/random_unitary.jl

```diff
@@ -0,0 +1,55 @@
+using MatrixAlgebraKit: qr_full!
+using Random: Random, AbstractRNG, randn!
+
+function square_zero_map(elt::Type, ax::Tuple{AbstractUnitRange,Vararg{AbstractUnitRange}})
+  return zeros(elt, (ax..., ax...))
+end
+
+# Imports `..` into the namespace.
+using EllipsisNotation
```

Comment:
This might not be used in this file? (also maybe we can move it to the top?)

Reply:
Good catch, I was using it before in a call to `fusedims` but rewrote it in a different way.